src/test/java/mate/academy/dao/RoleDaoTest.java

```diff
@@ -0,0 +1,59 @@
+package mate.academy.dao;
+
+import mate.academy.dao.impl.RoleDaoImpl;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.model.Role;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RoleDaoTest extends AbstractTest {
```

Comment:
```suggestion
package mate.academy.dao.imp;;

import mate.academy.dao.impl.RoleDaoImpl;
import mate.academy.exception.DataProcessingException;
import mate.academy.model.Role;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

public class RoleDaoImplTest extends AbstractTest {
```

a test should be in the same package that tested class
you're testing the concrete implementation of dao, so it should be shown in the test class name